src/sgame/sg_buildpoints.cpp

```diff
@@ -77,18 +77,19 @@ void G_UpdateBPVampire( int client ) // -1 to update everyone
 		return;
 	}
 
-	trap_SendServerCommand( client, va( "bpvampire %d %d", g_BPInitialBudgetHumans.Get(), g_BPInitialBudgetAliens.Get() ) );
+	trap_SendServerCommand( client, va( "bpvampire %d %d", static_cast<int>( level.team[ TEAM_HUMANS ].totalBudget ), static_cast<int>( level.team[ TEAM_ALIENS ].totalBudget ) ) );
```

Comment:
Maybe make a helper for this and call it in the callbacks for the g_InitialBudget* cvars? Then we don't need the updatevampirebp command at all

Reply:
The `updateBPVampire` command is not needed anymore once we merge this. Or so I believe.